codespell_lib/tests/test_dictionary.py

```diff
@@ -177,6 +177,7 @@ def test_error_checking_in_aspell(err, rep, err_aspell, rep_aspell, match):
     ('dictionary.txt', 'dictionary_rare.txt'),
     ('dictionary.txt', 'dictionary_usage.txt'),
     ('dictionary_rare.txt', 'dictionary_usage.txt'),
+    ('dictionary_rare.txt', 'dictionary_informal.txt'),
```

Comment:
Should we allow dictionary.txt and possibly the usage/informal combos now, or shall we add them as necessary?

Reply:
Skipping for later.